deploy/chart/values.yaml

```diff
@@ -156,6 +153,21 @@ roles:
           - list
           - patch
           - update
+      - apiGroups:
+          - apiextensions.k8s.io
+        resources:
+          - customresourcedefinitions
```

Comment:
Do we need this RBAC permissions on customresourcedefinitions?

Reply:
We don't need it. But since this is a revert, I think we can just keep it now and remove it in a later PR

Reply:
Exactly, not needed but it's a revert of the PR. We'll just re-open the PR and add the changes back with the fix.